activerecord/lib/active_record/enum.rb

```diff
@@ -248,7 +248,7 @@ def assert_valid_enum_definition_values(values)
           raise ArgumentError, error_message
         end
 
-        if values.is_a?(Hash) && values.keys.any?(&:blank?) || values.is_a?(Array) && values.any?(&:blank?)
+        if values.is_a?(Hash) && values.any? { |k, _v| k.blank? } || values.is_a?(Array) && values.any?(&:blank?)
```

Comment:
I don't think this is worth changing, since the new version is less readable and this line only runs when an enum is defined.